htdocs/compta/accounting-files.php

```diff
@@ -460,7 +472,19 @@
 	$log .= ','.$langs->transnoentitiesnoconv("Country");
 	$log .= ','.$langs->transnoentitiesnoconv("VATIntra");
 	$log .= ','.$langs->transnoentitiesnoconv("Sens")."\n";
-	$zipname = $dirfortmpfile.'/'.dol_print_date($date_start, 'dayrfc', 'tzuserrel')."-".dol_print_date($date_stop, 'dayrfc', 'tzuserrel').'_export.zip';
+	$zipname = $dirfortmpfile.'/'.dol_print_date($date_start, 'dayrfc', 'tzuserrel')."-".dol_print_date($date_stop, 'dayrfc', 'tzuserrel');
+	if(!empty($projectid)) {
+		$sql = 'SELECT t.ref FROM '.MAIN_DB_PREFIX.'projet as t';
```

Comment:
A better solution would be do to :

$project = new Project($db);
$project->fetch($projectid);
if ($project->ref) {
    $zipname .= '_'.$project->ref;
}

So there is no sql at all inside the php code.